ftp.go

```diff
@@ -74,11 +75,40 @@ type dialOptions struct {
 
 // Entry describes a file and is returned by List().
 type Entry struct {
-	Name   string
-	Target string // target of symbolic link
-	Type   EntryType
-	Size   uint64
-	Time   time.Time
+	EntryName string
```

Comment:
Renaming Name and Size is unnecessary since they are members of the `Entry` type and it reads nicely when using `entry.Name` and `entry.Size`